naughty/rhel-10/7716-checkpoint-restore-failure

```diff
@@ -0,0 +1,7 @@
+Traceback (most recent call last):
```

Comment:
The same things is happening in ubuntu-stable currently: https://github.com/cockpit-project/bots/issues/7692

We now have the criu debug dump in the test output, and we can match against it, like in  https://github.com/cockpit-project/bots/pull/7685/commits/3defe4a34fe0ffab95769e50f73f774780fcaee9. I think we should do this here, too.